react/week-19/lesson.md

```diff
@@ -135,8 +134,7 @@ Using JSX ([interactive version](http://jsbin.com/gekahexige/edit?html,output)):
 As you can see, this is much easier to read than both the straight `React.createElement` API and the vanilla JS API. Most people using React use JSX to write their components.
 
 > **Exercise B**
-> 1. Change the [JSX example from above](http://jsbin.com/gekahexige/edit?html,output) to instead render a `h1` with an id of `"main-header"` and the text "Hello Ali"
-> 2. Visit the [Babel REPL](https://babeljs.io/repl/) and copy/paste the `h1` that you just created
```

Comment:
Perhaps it's just me, but I do quite like the ability for students to see that JSX isn't "magic", it's just regular functions. I feel like it reduces the "this is weird new syntax" aspect of it.

Having said that I felt like we spent too much time on it in London for the last class. So I think I'm ok with removing this exercise.

Reply:
Right i understand, I think mentor can maybe just show to the class that actually JSX is just Javascript finally instead of having it as an exercise? What do you think? Could add it as a mentor note.

Reply:
I think adding it as a mentor note is a good idea 👍